src/edutap/wallet_apple/api.py

```diff
@@ -89,9 +105,13 @@ def create_auth_token(
     create an authentication token using cryptography.Fernet symmetric encryption
     """
     if fernet_key is None:
-        settings = Settings()
-        assert settings.fernet_key, "fernet_key is not set in the settings"
-        fernet_key = settings.fernet_key.encode("utf-8")
+        dynamic_settings_handler = get_dynamic_settings_handler()
+        if dynamic_settings_handler is not None:
+            fernet_key = dynamic_settings_handler.get_fernet_key()
```

Comment:
There is no criteria which can be used to lookup dynamic fernet key, so we should remove this setting from dynamic settings (we did the same in wallet_apple)